welcome.html

```diff
@@ -56,7 +56,10 @@
     }
 
     #details table{
-      margin: 1em auto;
+      margin: 0.5em auto;
+    }
```

Comment:
Quick fix: line spacing between classes and one space before opening braces: https://github.com/jsoxford/jsoxford.github.com/blob/develop/CONTRIBUTING.md#scss. :)